### PR TITLE
docs: #25  add Readme.md with description of project scripts, and setup (what to copy etc)

### PR DESCRIPTION
**Pull request by Ekaterina1994**
Add README.md with project description, create process.md with dev processes, update drawio with fb collections and business models

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Masters-way app 
+
+Multi-user application for tracking the educational process.
+
+### Modules
+* docs - project documentation
+* mw-webapp - front end for clients
```
Comment (tritonJS826) [inline README.md:7]: Who is client? Do we have frontend for developers or management?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+___
+
+### Requirements: 
+* pnpm: 7.25.0
```
Comment (tritonJS826) [inline README.md:11]: usually platform version is more important than package manager version

File: mw-webapp/src/model/businessModel/PlanForNextPeriod.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export class PlanForNextPeriod {
   public job: string;
 
   /**
-   * Enum @Time.unit (minute)
+   * Enum @Time.unit
```
Comment (tritonJS826) [inline mw-webapp/src/model/businessModel/PlanForNextPeriod.ts:19]: It is obvious from field type

**Review by tritonJS826 — APPROVED** [review]

File: docs/process.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+3. Create commit which name contains issue type, issue number with "#" symbol and short issue description. Example
+```docs: #25 Update UML diagram with firebase collections and business models```
+```docs: #25 Create Readme.md with project's description```
+
+3. Merge (or pull --rebase) changes from main and resolve conflicts
+
+4. Push branch to remote repository
+
+7. Create PR to main. PR name should contains issue number with "#" symbol and full issue name. Example
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply docs/process.md:17]: Done

File: README.md (hunk 1 above)
Comment (Ekaterina1994) [reply README.md:7]: Change to mw-webapp - front end

File: mw-webapp/src/model/businessModel/PlanForNextPeriod.ts (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/model/businessModel/PlanForNextPeriod.ts:19]: Change to Unit of time measurement

File: README.md (hunk 2 above)
Comment (Ekaterina1994) [reply README.md:11]: Add node version to requirements

**Review by SergioAJS — APPROVED** [review]

**Pull request merged**